### ISSUE TITLE
how to show back button ?

### ISSUE THREAD

**Issue by HusseinElGanzory** [bamlab/react-native-image-header-scroll-view#81]

**Comment by manuelwenner** [comment]
@HusseinElGanzory  I had / have the same question and I think you have to reimplement it by yourself. By default the default header is still visible. You can hide it by setting `headerShown: false`.

**Comment by HusseinElGanzory** [comment]
> @HusseinElGanzory I had / have the same question and I think you have to reimplement it by yourself. By default the default header is still visible. You can hide it by setting `headerShown: false`.

@manuelwenner  i solved it by creating  back button with position 'absolute' thanks

**Comment by manuelwenner** [comment]
Okay thanks for your comment. Was the default header always visible?

**Comment by HusseinElGanzory** [comment]
> Okay thanks for your comment. Was the default header always visible?

@manuelwenner yes it visible , and create absolute icon to show at the top of screen

**Comment by manuelwenner** [comment]
@HusseinElGanzory Which image type did you use for the back button?

**Comment by HusseinElGanzory** [comment]
> @HusseinElGanzory Which image type did you use for the back button?

i Used font awesome icon , through native base icon component
